app/src/main/res/layout/character_detail_layout.xml

```diff
@@ -12,76 +13,66 @@
         app:layout_constraintTop_toTopOf="parent"
```

Comment:
If you are using a LinearLayout this attributes are not needed anymore 
`app:layout_constraintEnd_toEndOf="parent"
 app:layout_constraintTop_toTopOf="parent"`, remove it please.

Reply:
Done thx